src/scripts/generator_base.py

```diff
@@ -69,10 +72,7 @@ def create_false_numbers_array_string(self, false_values_count):
     def create_declarations_string(self, declarations_dictionary):
         declaration_string = ""
         for key, value in declarations_dictionary.items():  # For E.g.: a0 = 38473837;
-             if isinstance (value, str): # Quote string values
-                 declaration_string += "%s = '%s';\n" % (str(key), str(value))
-             else:
-                 declaration_string += "%s = %d;\n" % (str(key), value)
+             declaration_string += "%s = %s;\n" % (str(key), str(value))
```

Comment:
This utility function is changes over and over the PRs. This should be finalized somehow.